<launch>
    <arg name="port" default="/dev/ttyUSB0" />
    <arg name="budrate" />
    <node pkg="gps_driver" type="rtk_driver.py" name="gps_driver" output="screen">
        <param name="port" value="$(arg port)" />
    </node>
</launch>

